### DISCUSSION TITLE
Import issue

### DISCUSSION THREAD

**Discussion by ndaskalovic** [davidwickerhf/instaclient#11] [Q&A] [answered]
im having this issue: ImportError: cannot import name 'InstaClient' from 'instaclient' (/usr/local/lib/python3.9/site-packages/instaclient/__init__.py)

could you help me out?

**Comment by davidwickerhf** [comment, marked as answer]
Hi @ndaskalovic , I'm so sorry for the late reply to your question!
I rarely checked the Discussions tab as I didn't think someone would use it as of yet...
I believe that issue has been fixed with the new version v2.7.24, so you can update with ` pip install -U instaclient`
when updating, I would pay attention to a couple of things:
- arguments in the functions client.get_followers, client.get_notifications and client.get_profile


If the error persists, than you can solve the issue by using the following instead:
`from instaclient.client.instaclient import InstaClient`
